src/Components/Patient/PatientInfoCard.tsx

```diff
@@ -140,6 +140,30 @@ export default function PatientInfoCard(props: {
     prefetch: !!consultation?.treating_physician_object?.username,
   });
 
+  const { data: consentRecords, loading: consentRecordsLoading } = useQuery(
+    routes.listConsents,
+    {
+      pathParams: {
+        consultationId: consultation?.id ?? "",
+      },
+      prefetch: !!consultation?.id,
+    },
+  );
+
+  const { data: consentFiles, loading: consentFilesLoading } = useQuery(
+    routes.viewUpload,
+    {
+      query: {
+        file_type: "CONSENT_RECORD",
+        associating_id: consentRecords?.results.map((cr) => cr.id).join(","),
+        limit: 1000,
```

Comment:
if we are just fetching this see if there are records are not, can we just use 1 here